docs/admin/auth.md

```diff
@@ -37,6 +37,11 @@ CODER_OAUTH2_GITHUB_ALLOWED_ORGS="your-org"
 CODER_OAUTH2_GITHUB_CLIENT_ID="8d1...e05"
 CODER_OAUTH2_GITHUB_CLIENT_SECRET="57ebc9...02c24c"
 ```
+**Note:** To allow everyone to signup using GitHub, set
+```console
+CODER_OAUTH2_GITHUB_ALLOW_EVERYONE=true
+CODER_OAUTH2_GITHUB_ALLOWED_ORGS=
```

Comment:
We don't need to explain this as it's not required to set allowed_orgs= empty (omitting is preferred). It's documented in the application options and you will receive an informative error if you try to set orgs/teams as well.

```suggestion
```